media-gfx/pqiv/pqiv-2.12.ebuild

```
@@ -0,0 +1,71 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit edo linux-info toolchain-funcs xdg-utils
+
+SRC_URI="https://github.com/phillipberndt/pqiv/archive/${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+
+DESCRIPTION="A powerful GTK 3 based command-line image viewer with a minimal UI"
+HOMEPAGE="https://github.com/phillipberndt/pqiv http://www.pberndt.com/Programme/Linux/pqiv/"
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="archive ffmpeg imagemagick pdf postscript webp"
```

The order of variables here isn't in the defined order:

```suggestion

DESCRIPTION="Powerful GTK 3 based command-line image viewer with a minimal UI"
HOMEPAGE="https://github.com/phillipberndt/pqiv http://www.pberndt.com/Programme/Linux/pqiv/"
SRC_URI="https://github.com/phillipberndt/pqiv/archive/${PV}.tar.gz -> ${P}.tar.gz"

LICENSE="GPL-3+"
SLOT="0"
KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
IUSE="archive ffmpeg imagemagick pdf postscript webp"
```